<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>.AppleSystemUIFont</family>
    <pointsize>18</pointsize>
    <kerning>false</kerning>
   </font>
  </property>
  <property name="windowTitle">
   <string>StudentHub</string>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>40</y>
      <width>701</width>
      <height>191</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>18</pointsize>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="academicButton">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>260</y>
      <width>681</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>.AppleSystemUIFont</family>
      <pointsize>17</pointsize>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="text">
     <string>Information on majors, looking up classes, advisement, and the articulation agreements 
BMCC has with other colleges.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="activitiesButton">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>350</y>
      <width>681</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>.AppleSystemUIFont</family>
      <pointsize>21</pointsize>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="text">
     <string>Clubs, Success Programs, and Directory</string>
    </property>
   </widget>
   <widget class="QPushButton" name="resourcesButton">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>440</y>
      <width>681</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>.AppleSystemUIFont</family>
      <pointsize>21</pointsize>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="text">
     <string>Student resources</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
